### ISSUE TITLE
Integration between did & dataverse for did codes & projects

### ISSUE THREAD

**Issue by jenunn** [Puzzlepart/did#1191]
- [x] I have searched for existing issues to avoid creating duplicates

### TLDR
In order for Puzzlepart CRM & did to be in sync over the available DID codes in both systems, we want to implement an integration between the 2 systems.

In order to do this we have tried accessing the Cosmos DB using the connector "Azure Cosmos DB" connector in Power Automate.
This returned an error saying "Sql api is not supported for this database account
ActivityId: cc482c5c-39a6-4cf2-b811-15fb0d8f999c, Microsoft.Azure.Documents.Common/2.14.0"

We should instead use a direct call using graphQL request that needs to be transformed into a valid REST request because Power Automate does not have capabilities to use graphQL.

### Description
* What is the feature?
An integration between Dataverse & did for having did codes & projects up to speed in both systems
* Why should we add this feature?
To be able to easily select did codes in Dataverse & did no matter which system the did codes & projects are created in.
* How does this improve the product?
Easier user experience for users of did & Puzzlepart CRM

We could potentially also trigger an Azure Runbook which has a webhook we can trigger from a Power Automate that can handle the GraphQL queries towards did.

Potentially 3 automations:

1. Create a Power Automate Flow that triggers upon creation of a new did code or did project. This flow can then trigger an Azure runbook that can execute the GraphQL queries to the custom did API's.
This will take care of the sync of creations from Dataverse to did.

2. Create a Power Automate Flow or Azure Runbook that triggers on a recurring interval that can retrieve data from both systems and make sure to update/create those items that is missing or is not in sync
This will take care of the sync of data between both systems.

3. Create something that can trigger on new rows in Cosmos DB where a new did project or did code has been created from the did webapp and automatically sync this to Dataverse.
This will take care of the sync of creations from did to Dataverse
